### DISCUSSION TITLE
[Feature] 关于csv格式的大文件导入的速度优化

### DISCUSSION THREAD

**Discussion by islvfz** [dream-num/univer#2304] [Q&A | 常见问题]
### 初始清单

- [X] 这真的是个问题吗？
- [X] 我已经在 Github Issues 中搜索过了，但没有找到类似的问题。

### 问题

使用官方的demo，导入15M的csv格式的文件（大概300万的单元格），用了40秒。希望能减少导入文件的时间。

### 解决方案

除了预处理csv文件数据这种方式外，是否有别的方式优化导入速度

**Comment by jikkai** [comment]
This does not appear to be a feature request, so I have converted it to a discussion.
